pkgs/tools/misc/limitcpu/default.nix

```diff
@@ -0,0 +1,27 @@
+{lib, stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  pname = "limitcpu";
+  version = "2.7";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/limitcpu/${pname}-${version}.tar.gz";
```

Comment:
That link is not working anymore

```
       >
       > trying https://kent.dl.sourceforge.net/sourceforge/limitcpu/limitcpu-2.7.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0   308    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       >   0   154    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download limitcpu-2.7.tar.gz from any mirror
       For full logs, run 'nix log /nix/store/bklgi1cqfdhxx8yp158v7zg1d7cm33j6-limitcpu-2.7.tar.gz.drv'.
```

Reply:
Ah, right. Since I renamed it cpulimit->limitcpu.